bleak/backends/bluezdbus/manager.py

```diff
@@ -843,15 +844,21 @@ def _run_advertisement_callbacks(
             callback(device_path, device.copy())
 
 
+_instances: WeakKeyDictionary[Any, BlueZManager] = WeakKeyDictionary()
+
+
 async def get_global_bluez_manager() -> BlueZManager:
     """
-    Gets the initialized global BlueZ manager instance.
+    Gets an existing initialized global BlueZ manager instance associated with the current event loop,
+    or initializes a new instance.
     """
 
-    if not hasattr(get_global_bluez_manager, "instance"):
-        setattr(get_global_bluez_manager, "instance", BlueZManager())
-
-    instance: BlueZManager = getattr(get_global_bluez_manager, "instance")
+    loop = asyncio.get_event_loop()
```

Comment:
`get_event_loop()` is deprecated, this should be:

```suggestion
    loop = asyncio.get_running_loop()
```